extensions/interactions/SetInput/SetInput.py

```diff
@@ -22,9 +22,11 @@ class SetInput(base.BaseInteraction):
 
     name = 'Set Input'
     description = 'Allows learners to enter an unordered set of strings.'
-    display_mode = base.DISPLAY_MODE_SUPPLEMENTAL
+    display_mode = base.DISPLAY_MODE_INLINE
```

Comment:
I'm starting to question whether 'inline' is actually a good description for these types of interactions. I'm not so sure the opposite of supplemental is inline. Inline indicates it fits within one line, but that's not the case for set input. It cannot be summarized into one line, yet it can fit on the main content card and does not require a supplemental card. What are your thoughts on this?